app/src/main/java/com/devshish/internship/presentation/ui/player/PlayerViewModel.kt

```diff
@@ -56,6 +74,10 @@ class PlayerViewModel(
         _playbackPosition.value = convertMillisToTime(progress)
     }
 
+    fun onLyricsClick(searchSong: SearchSong) {
```

Comment:
Also, here you can make onLyricsClick without parameters, and get song to search lirycs from private livedata searchSong